backend/routes/macroRoutes.js

```
@@ -0,0 +1,709 @@
+        if (!settingsValidation.valid) {
+            return res.status(400).json({
+                success: false,
+                error: '설정이 유효하지 않습니다',
+                details: settingsValidation.errors
+            });
+        }
+
+        // URL 접근성 테스트 (HEAD 요청)
+        try {
+            const axios = require('axios');
+            const response = await axios.head(targetUrl, {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)